### DISCUSSION TITLE
Unable to remove reviewers occasionally

### DISCUSSION THREAD

**Discussion by devversion** [community/community#69208] [Pull Requests]
### Select Topic Area

Bug

### Body

Hi team,

Regularly on GitHub we are seeing issues where a pending reviewer cannot be removed from a pull request (note: we do not use codeowners). When the UI combobox for reviewers opens, we can uncheck the reviewer, but upon closing the panel, the reviewer is **not removed**. This can be tried repeatedly but never ends up working.

**Reproduction Steps**

Here is an example where alxhub@ cannot be removed: https://github.com/angular/angular/pull/51807

Unclear how it can be reproduced. It happens randomly. I suspect somehow the internal state is messed up when a PR is created, reviewers are requested by a bot (and then sometimes removed by the user via UI again where the  bot re-adds reviewers instantly).

Here is a video:

https://github.com/community/community/assets/4987015/ecac37eb-248d-46eb-95f6-113b5cae851c

The devtools show an error when the reviewers are removed and the panel closes. Somehow a 404 is returned.

![image](https://github.com/community/community/assets/4987015/7db59041-5f11-46f1-b6d4-528c5aa530ca)

Trying to remove the reviewer via REST API does not work either.

**Fault Type**
Permanent - Occurring repeatably

**Blocking?**
Yes, with strict reviewer policies, this can block merging PRs

**Comment by AceCoderLaura** [comment]
I've had this same issue on a PR that was closed two years ago. It's been stuck on my review-requested:@me list for years, it would be nice to get rid of it.

**Comment by j18ter** [comment]
This continues to be annoying for me too. How can we get this raised in priority to see some action?

**Comment by HughWarrington** [comment]
For me this is not occasionally, it is every time I am initially creating a PR.

**Comment by colinhowe** [comment]
+1 I'm starting to see this from time to time. Really frustrating as reviews just start piling up :(

**Comment by davidkominekhypixelstudios** [comment]
Also happening for me.
Specifically when i use `gh pr edit` from GitHub Action workflow to add reviewer to PR.

Repro:
1) have open draft PR
2) run GitHub Action workflow with `gh pr edit <url> --add-reviewer some_user`
3) try to remove `some_user` from reviewers list manually via UI
- seeing same behavior as in the video above
4) try to remove `some_user` from reviewers list via GitHub Action workflow: `gh pr edit <url> --remove-reviewer some_user`
- getting error `GraphQL: Could not add requested reviewers to pull request. (requestReviews)`

Note:
GitHub Action workflow is using token with Pull request Write permission for the `gh pr edit` calls

**Comment by JoshCarrollAcima** [comment]
Any chance this has been answered in another thread? I see the exact same thing... it seems to happen with 1 or 2 Github users, but not all.

**Comment by jdkworld** [comment]
This is also happening for me occasionally. Not only while I'm creating a PR, but also on already existing PRs. I'm trying to remove reviewers in the interface, just as the screen recording in the description of this issue shows.
